docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -110,9 +110,84 @@ spec:
     - url: "github.com/kyma-project/addons//addons/index.yaml"
     # bitbucket.org URL with no prefix. It is automatically interpreted as a Git repository source.
     - url: "bitbucket.org/kyma-project/addons//addons/index.yaml"
+    # Git SSH protocol with a reference to a secret containing base64 encoded SSH private key
+    - url: "git::ssh://git@github.com/kyma-project/private-addons.git//addons/index.yaml?sshkey={key}"
+      secretRef:
+        name: auth
+        namespace: stage
 ```
 
->**NOTE:** For now, the SSH protocol is not supported.
-
   </details>
 </div>
+
+## Authorization
```

Comment:
maybe we can mention somewhere that we have the general templating allowed in URL field. and basically, we are using that for handling the auth. but you can place there also the whole URL 
```
    # HTTPS protocol with basic authorization provided.
    - url: "{URL}"
      secretRef:
        name: auth
        namespace: stage 
```

I cannot find direct info that *key* used in url must match key in secret